torchao/float8/float8_tensor.py

```diff
@@ -47,31 +45,41 @@
 #    to configure all three gemms, also not user facing
 
 
-# ScaledMMConfig is a namedtuple that defines the configuration for the scaled_mm in the forward and backward pass.
-# emulate: whether to emulate the matmuls in fp32
-# use_fast_accum: whether to use the fast-accumulation option for scaled_mm
-# fp8_output: whether to output the result of the scaled_mm in fp8
-# pad_inner_dim: whether to pad the inner dimension of a and b with 0s. This is needed for matmuls not aligned to 16.
-ScaledMMConfig = namedtuple(
-    "ScaledMMConfig",
-    ["emulate", "use_fast_accum", "fp8_output", "pad_inner_dim"],
-    defaults=[False, False, False, False],
-)
+class ScaledMMConfig(NamedTuple):
```

Comment:
if this is the object to make public, can we move it to `config.py` to keep this file all-private?

alternatively, how about just creating this from a different public config, in case that works better with torchao inference code?